### PR TITLE
Add DNS Provider for otc

### PR DESCRIPTION
**Pull request by cgroschupp**
Add DNS Provider for the german Open Telekom Cloud (OTC).

https://cloud.telekom.de/infrastruktur/open-telekom-cloud/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: go
 go:
-- 1.6.3
 - 1.7
+- 1.8
```
Comment (mholt) [inline .travis.yml]: This change is good, but unrelated to this PR. Can you please splice it out into a different branch?

Comment (cgroschupp) [inline .travis.yml]: Was actually my plan. Thank you for the quick merge. What do you think about this pull request? Is there anything I need to do?

**Comment by cgroschupp** [comment]
@xenolf @mholt Is it possible that one of you review this pr?

**Comment by mholt** [comment]
Without knowing anything about OTC, the lego integration looks OK to me. Will merge in a few days or at next ping if I don't hear anything back from Sebastian.

**Pull request merged**